backend/routes/indexer/utils.go

```diff
@@ -0,0 +1,25 @@
+package indexer
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+)
+
+func combineLowHigh(tokenIdLowHex, tokenIdHighHex string) (*big.Int, error) {
+	// Remove leading zeros from the hex strings
+	tokenIdLowHex = strings.TrimLeft(tokenIdLowHex, "0")
```

Comment:
Please change all variable naming away from tokenId now that the function is generic